packages/ui/WizardForm.tsx

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from "next/router";
+
+import Stepper from "./Stepper";
+
+type DefaultStep = {
+  title: string;
+  description: string;
+  content: JSX.Element;
+  enabled?: boolean;
+};
+
+function WizardForm<T extends DefaultStep>(props: { href: string; steps: T[] }) {
```

Comment:
Moved this component from `console` to `ui`, after this PR I will change`console` to use it from `ui` too.